index.html

```diff
@@ -7,40 +7,75 @@
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
   <style>
     .header {
-      text-align: center;
+      text-align: left;
```

Comment:
You're really letting loose with the styles there. 😜 (Don't worry, I see that you've added the table-hover class below as a second style change.)

I'd push you to use these early weeks to try things out, make mistakes, get messy. Start trying to find a unique look and feel that you're proud of. This isn't a design class, but when it comes to the final project, it'll make the difference between a satisfactory class assignment and something that you can show off to the wider world.